CODEOWNERS

```diff
@@ -14,6 +14,12 @@
 * @cilium/janitors
 /CODEOWNERS @cilium/contributing
 /.github/ @cilium/contributing
+/.github/cilium-cli-test-job-chart/ @cilium/ci-structure
+/.github/gcp-vm-startup.sh @cilium/ci-structure
+/.github/get-kubeconfig.sh @cilium/ci-structure
+/.github/in-cluster-test-scripts/ @cilium/ci-structure
```

Comment:
@nebril Which of these need to be owned by cilium/maintainers due to potential security impact?

Reply:
`cilium-cli-test-job-chart` should probably be owned by maintainers, I can imagine a scenario when templating helm allows to leak secrets
`gcp-vm-startup.sh` runs in gcp vm, so it should be safe
`get-kubeconfig.sh` should be owned by maintainers
`in-cluster-test-scripts.sh` are sent to cluster and executed there, so probably doesn't need maintainers review.

Reply:
Thanks! :pray: 

I added @cilium/github-sec to the security-sensitive entries and added you as a co-author.